src/Core/UbuntuDriversBackend.vala

```diff
@@ -85,9 +84,13 @@ public class AppCenterCore.UbuntuDriversBackend : Backend, Object {
                 continue;
             }
 
-            // Filter out the nvidia server drivers
-            if (package_name.contains ("nvidia") && package_name.contains ("-server")) {
-                continue;
+            // Filter NVIDIA packages from Ubuntu
+            if (package_name.contains ("nvidia")) {
+                var name = package_name.split (",")[0];
+                if (!yield packaged_by_pop(null, name)) {
+                    stderr.printf("%s not packaged by Pop\n", name);
```

Comment:
Wont this make pop shop print an error every time it checks for packages?

Reply:
If attached to a terminal, it'd print on stderr